languages/ruby/exercises/concept/floating-point-numbers/.docs/introduction.md

```diff
@@ -0,0 +1,12 @@
+A floating-point number is a number with zero or more digits behind the decimal separator. Examples are `4.0`, `0.1`, `3.14`, `-6.4` `16.984025` and `1024.0`.
```

Comment:
Could you also add the information in this file to the `after.md` document?

Reply:
I will, thanks for mentioning it